.dockerignore

```diff
@@ -2,8 +2,30 @@
 **/data
 **/npm-debug.log
 **/.git
-**/.env
-**/.env.*
+
+**/.env.local
+**/.env.test
+**/.env.*.local
+**/.env.*.test
+
+apps/deploy-web/.env
+apps/deploy-web/.env.*
+
+apps/indexer/.env
+apps/indexer/.env.*
+
+apps/landing/.env
+apps/landing/.env.*
+
+apps/provider-console/.env
+apps/provider-console/.env.*
+
+apps/provider-proxy/.env
+apps/provider-proxy/.env.*
+
+apps/stats-web/.env
+apps/stats-web/.env.*
```

Comment:
Could we do this instead to include all apps and prevent an oversight when creating new apps?
```
apps/*/.env
apps/*/.env.*
```

Reply:
Same for .gitignore if it makes sense

Reply:
I need these to be ignored for all apps except for api now. Eventually when all apps are loading envs in the same way this can be patterned